blob/blob.go

```diff
@@ -1422,6 +1423,13 @@ type WriterOptions struct {
 	// asFunc converts its argument to driver-specific types.
 	// See https://gocloud.dev/concepts/as/ for background information.
 	BeforeWrite func(asFunc func(any) bool) error
+
+	// IfNotExist is used for conditional writes. When set to true, if
+	// a blob exists for the same key in the bucket, the write operation
+	// won't take place.
+	// It only works for AWS S3, Google Storage and Azure. Setting this to
```

Comment:
This generally isn't OK; this is supposed to be a portable library. Can you implement this option for fileblob and memblob too? It shouldn't be too hard.

Reply:
Sure! Will do!